docs/imageomics/treeoflife10m.md

```diff
@@ -1,9 +1,34 @@
-# Dataset Card for TreeOfLife-10M
+# How to Create TreeOfLife-10M
 
+**Note:** [TreeOfLife-10M](https://huggingface.co/datasets/imageomics/TreeOfLife-10M) has the EOL images, but not iNat21 or BIOSCAN-1M due to licensing restrictions. To reconstruct the _full_ dataset, please follow the steps outlined below in Reproduce TreeOfLife-10M.
 
-## How to Create TreeOfLife-10M
+## Reproduce TreeOfLife-10M
 
-This is the process for creating the entire dataset, version 3.3 (which we used to train BioCLIP for the public release).
+1. **Download [TreeOfLife-10M](https://huggingface.co/datasets/imageomics/TreeOfLife-10M)**:
+   - Download the tar files and metadata from [TreeOfLife-10M](https://huggingface.co/datasets/imageomics/TreeOfLife-10M) on Hugging Face, and download [iNat21](https://github.com/visipedia/inat_comp/tree/master/2021#data) and [BIOSCAN-1M](https://zenodo.org/doi/10.5281/zenodo.8030064) into `data/TreeOfLife-10M/dataset/`, as specified in [`disk_reproduce`](/src/imageomics/disk_reproduce.py).  
```

Comment:
This will be automated with `scripts/reproduce_tree-of-life-10m.sh` with `cd scripts && bash reproduce_tree-of-life-10m.sh`. 
I can rename this to something narrower like `scripts/download_tree-of-life-10m_components.sh`

Reply:
Yes. I just added a dynamic path so someone can run the script from anywhere on the filesystem and it'll still put the dataset in the default location of `../data/TreeOfLife-10M` relative to the script.

We might add another note to the instructions on this step that the user may edit the `dataset_path=` line of the script if they prefer another location for their data, or that they should clone this repository directly into a location that's capable of storing the full dataset.

e.g. they should not clone this into their home directory and use the default location.